API.md

```diff
@@ -755,6 +755,8 @@ The events emitted by the Time API are:
     If `tick` is false,then the bounds change was not due to an automatic tick, 
     and a query for historical data may be necessary, depending on your data 
     caching strategy, and how significantly the start bound has changed.
+* `tick`: emitted whenever an active clock ticks. The callback will be invoked
```

Comment:
Don't need a new event for this when `bounds` carries a second argument to determine whether or not it was a clock `tick`.